build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mkdir build
```

Comment:
```suggestion
mkdir -p build
```
(will not fail in case of existing build dir)